simulators/backend/__init__.py

```diff
@@ -1,233 +1,18 @@
 from SocketServer import ThreadingTCPServer
-from simulators.common import ListeningSystem
-from simulators.backend import grammar
-from simulators.backend.genericbackend import GenericBackend, BackendException
-from simulators.backend.sardara import Sardara
-from simulators.utils import ACS_TO_UNIX_TIME
+from simulators.utils import get_multitype_systems
+from simulators.common import MultiTypeSystem
 
 
+systems = get_multitype_systems(__file__)
 servers = [
-    (('0.0.0.0', 12800), (), ThreadingTCPServer, {'backend_type': Sardara})
+    (('0.0.0.0', 12801), (), ThreadingTCPServer, {'system_type': 'sardara'}),
+    (('0.0.0.0', 12802), (), ThreadingTCPServer, {'system_type': 'mistral'}),
```

Comment:
Whenever I try to launch the simulator with the following command:
`discos-simulator -s backend start`
This error pops up:
`ValueError: System type mistral not found.`
Which is actually correct since the mistral implementation is not yet written.
Also, if I launch the following command:
`discos-simulator -s backend -t sardara start`
I get 2 instances of Sardara type. Not sure if this is the correct behavior

Reply:
Yes, that is right, I did not add the mistral.py file to this commit. I remove mistral from the servers

Reply:
About the two processes, I think it it normal because each simulator is executed in a separate process.

Reply:
@giuseppe-carboni  The *multiple instances* problem will be fixed in issue https://github.com/discos/simulators/issues/267 .